### ISSUE TITLE
Model flow problem

### ISSUE THREAD

**Issue by lhy0322** [khanhlee/bert-enhancer#7]
cv represents training set and ind represents test set，What does cv_neg 、ind_neg 、ind_pos and cv_pos mean?
Is it to convert all training sets and test sets into JSON by using Bert, and then into CSV, and then put them all into CNN?

**Comment by khanhlee** [comment]
> cv represents training set and ind represents test set，What does cv_neg 、ind_neg 、ind_pos and cv_pos mean?
> Is it to convert all training sets and test sets into JSON by using Bert, and then into CSV, and then put them all into CNN?

1. because our problem is binary classification, I separated into two sets (positive and negative), thus the files included "neg" and "pos"
2. yes, we converted to CSV and put to neural network
